### PR TITLE
BaseAR now catches errors in __isset

### PR DESCRIPTION
**Pull request by SamMousa**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #16245

### CODE REVIEW

**Comment by samdark** [comment]
Travis failed.

**Comment by SamMousa** [comment]
I forgot something :)

**Comment by SamMousa** [comment]
Still hhvm failing just like in the other PR...

Not sure why

**Comment by SamMousa** [comment]
I don't suppose we could drop hhvm starting in 2.0.16 :dagger: ?

**Comment by samdark** [comment]
Restarted it just in case...

**Comment by SamMousa** [comment]
@samdark Travis build succeeded!

**Comment by samdark** [comment]
Need a changelog then.

**Comment by SamMousa** [comment]
Done

File: .travis.yml [hunk 1]
```diff
@@ -91,8 +91,9 @@ matrix:
         - mysql
         - postgresql
 
-    # test against the latest HHVM version by using a newer image
-    - php: hhvm
+    # test against the latest pre 3.26 HHVM version by using a newer image.
+    # @see https://github.com/facebook/hhvm/issues/8192
+    - php: hhvm-3.24
```
Comment (samdark) [inline .travis.yml]: They've fixed it: https://github.com/facebook/hhvm/issues/8192

**Comment by SamMousa** [comment]
https://github.com/facebook/hhvm/issues/8196

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**